pyvista/core/filters/data_set.py

```diff
@@ -4899,37 +4892,37 @@ def compute_derivative(
             derivative quantities.  Defaults to the active scalars in
             the dataset.
 
-        gradient : bool, str, optional
+        gradient : bool, str, default: True
```

Comment:
Should all these below parameters have `bool | str` instead of `bool, str`?

Reply:
Resolved in 367d26a111e306d978c835b03f1441d8472fe96d.